src/screens/FeedScreen/Content.js

```diff
@@ -318,7 +318,7 @@ const Content = ({
             fontSize={normalizeFontSizeByWidth(14)}
             text={message}
             topicContainer={{
-              position: item.post_type === POST_TYPE_STANDARD ? 'absolute' : 'relative',
+              position: 'absolute',
               bottom: 0
             }}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from conditional to absolute positioning could potentially cause layout issues depending on the post type. If there was a specific reason for the previous condition (i.e., different post types required different positioning), this change might introduce bugs. Please ensure that this change is thoroughly tested with different post types.
```diff
- position: item.post_type === POST_TYPE_STANDARD ? 'absolute' : 'relative',
+ position: 'absolute',
  bottom: 0
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->